### DISCUSSION TITLE
[Help Wanted] Using the Super editor, is it possible to have a serif font?

### DISCUSSION THREAD

**Discussion by red-ochre** [standardnotes/forum#3585] [Suggestions] [answered]
**Describe the issue**
I like the new Super Editor and I also prefer a serif font but I can't see a way to choose a different font. Is there a way?

**Comment by effieeee** [comment, marked as answer]
Hi @red-ochre, unfortunately there aren't any other font options for the Super note, apart from the **Monospace** font option in the Appearance settings of the app. We'll try to keep this in mind as a feature request though 🥲
